assets/src/data/persistence/extrinsic.ts

```diff
@@ -0,0 +1,87 @@
+import { SectionSlug } from 'data/types';
+import { makeRequest } from './common';
+
+export type ExtrinsicRead = Object;
+export type ExtrinsicUpsert = {
+  result: 'success',
+};
+export type ExtrinsicDelete = {
+  result: 'success',
+};
+
+export function read_global(keys: string[] | null = null) {
```

Comment:
I'm surprised the linter didnt complain here, but our ts coding standards say to use camelCase instead of snake_case for functions https://github.com/airbnb/javascript#naming--camelCase